<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1079</width>
    <height>436</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>11 - Morphing - Rados≈Çaw Mariowski</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QFrame" name="frmCanvas">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>280</width>
      <height>366</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QFrame" name="frmCanvas2">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>10</y>
      <width>280</width>
      <height>366</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QCheckBox" name="chkVisibility">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>30</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Hide Markers</string>
    </property>
   </widget>
   <widget class="QSlider" name="sldMorph">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>390</y>
      <width>1051</width>
      <height>31</height>
     </rect>
    </property>
    <property name="maximum">
     <number>20</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="btnPlay">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>70</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Play</string>
    </property>
   </widget>
   <widget class="QFrame" name="frmCanvas3">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>10</y>
      <width>280</width>
      <height>366</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
